content/kubermatic/master/upgrading/2.14_to_2.15/_index.en.md

```diff
@@ -5,7 +5,384 @@ weight = 90
+
+**Helm 2**
+
+```bash
+helm --tiller-namespace kubermatic-installer delete --purge kubermatic
+```
+
+Like we did on the master, apply the updated CRDs now:
+
+```bash
+kubectl apply -f charts/kubematic/crd/
+```
+
+The seed cluster is now ready to be managed by the operator. On the master cluster, update the Seed resource
+for the seed cluster and remove the `operator.kubermatic.io/skip-reconciling` annotation. Once the Seed is
+updated, the operator will start to reconcile the seed cluster. It is recommended to follow the operator's
+logs to spot any problems.
+
+After all seed clusters have been migrated, the operator migration is finished and for health and safety reasons,
+everyone involved should take a break.
+
+## Deprecation of `nodeport-proxy` Helm Chart
+
+In conjunction with the operator being promoted to the recommended installation method, we also deprecate the
+old `nodeport-proxy` Helm chart. The proxy is still a required component of any Kubermatic setup, but is now
+managed by the Kubermatic Operator (similar to how it manages seed clusters).
+
+The migration to the operator-managed nodeport-proxy is relatively simple. The operator by default creates the
+new nodeport-proxy inside the Kubermatic namespace (`kubermatic` by default), whereas the old proxy was
+living in the `nodeport-proxy` namespace. Due to this, no naming conflicts can occur and in fact, both proxies
+can co-exist in the same cluster.
+
+The only important aspect is where the DNS record for the seed cluster is pointing. To migrate from the old
+to new nodeport-proxy, all that needs to be done is switch the DNS record to the new LoadBalancer service. The
+new services uses the same ports, so it does not matter what service a user is reaching.
+
+To migrate, find the new LoadBalancer service's public endpoint:
+
+```bash
+kubectl -n kubermatic get svc
+#NAME                   TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)                                          AGE
+#kubermatic-api         NodePort       10.47.248.61    <none>          80:32486/TCP,8085:32223/TCP                      216d
+#kubermatic-dashboard   NodePort       10.47.247.32    <none>          80:32382/TCP                                     128d
+#kubermatic-ui          NodePort       10.47.240.175   <none>          80:31585/TCP                                     216d
+#nodeport-proxy         LoadBalancer   10.47.254.72    34.89.181.151   32180:32428/TCP,30168:30535/TCP,8002:30791/TCP   182d
+#seed-webhook           ClusterIP      10.47.249.0     <none>          443/TCP                                          216d
+```
+
+Take the `nodeport-proxy`'s EXTERNAL IP, in this case `34.89.181.151`, and update your DNS record for
+`*.<seedname>.kubermatic.example.com` to point to this new IP.
+
+It will take some time for the DNS changes to propagate to every user, so it is recommended to leave the old
+nodeport-proxy in place for a period of time (e.g. a week), before finally removing it:
```

Comment:
Probably recommending 72h should be enough:

https://ns1.com/resources/dns-propagation#:~:text=DNS%20propagation%20is%20the%20time,typically%20takes%20a%20few%20hours.

Reply:
Well apparently we should add 10h to this to be sure that each cluster resource is reconciled and `advertise-address` KAS flag is updated, here we could optimise a bit maybe

- [ ]

Reply:
There might be endusers that have weird DNS problems and use manually managed hostsfiles, so KKP admins will probably want to give their users (endusers) ample time to migrate their systems.

Basically the urgeny in removing the old nodeport-proxy is dictacted by the costs the unused old LoadBalancer might incur. Probably not a lot.